README.md

```diff
@@ -9,9 +9,9 @@ This sample library requires Python 3.7+. You can download Python [here](https:/
 
 ## About the library
 
-The python OCS library is an introductory language-specific examples of programming against OSISoft Cloud Services ([OCS](https://www.osisoft.com/Solutions/OSIsoft-Cloud-Services/)). It is as instructional samples only and are not for production use.
+The python ADH library is an introductory language-specific example of programming against Aveva Data Hub. It is intended as instructional samples only and are not for production use.
```

Comment:
As a replacement until a page like that exists, we could potentially use this https://www.aveva.com/content/dam/aveva/documents/datasheets/Datasheet_AVEVA_DataHubAnnouncementand_FAQ_10-21.pdf

or just keep the OCS for now.

Reply:
i think that link is fine